build.xml

```diff
@@ -132,7 +132,7 @@ oracle.jar=C:/workspace/HEAD/for_development/bisdevsrv28/jboss/server/infra/lib/
         <property name="compile.dir" value="${output.dir}/classes" />
         <property name="exposed.dir" value="${output.dir}/exposed" />
         <property name="gensrc.dir" value="${output.dir}/gensrc" />
-        <property name="dist.dir" value="/jython" />
+        <property name="dist.dir" value="${work.dir}/dist" />
```

Comment:
this was intentional...otherwise YOUR local dev path will appear in stacktraces

Reply:
got it, I'll revert this, just changed it to get it to build in a relative path